Editor/Scripts/DCCIntegration/Integrators/MayaIntegrator.cs

```diff
@@ -107,6 +107,10 @@ internal class MayaIntegrator : BaseDCCIntegrator {
         string arg = string.Format(argFormat, loadPluginCmd+FINALIZE_SETUP);
         bool setupSuccessful = SetupAutoLoadPlugin(dccToolInfo.AppPath, arg);
 
+        if (setupSuccessful) {
+            var analyticsClient = MeshSyncAnalyticsFactory.CreateAnalytics();
+            analyticsClient.UserInstalledPlugin("maya");
+        }
```

Comment:
Since this is exact same for each dcc integrator except for he string that gets set, could this be handled once in the base integrator?

Reply:
Given the amount of code, I didn't think it was worth it especially considering I'm highly skeptical of the object model in place for BaseDCCIntegrator actually adding any value

Reply:
It's a small amount now, but we could easily want to add more to this at some point. Especially as there's already a base class in place that could abstract this.

Reply:
The correct improvement here would be to factor out the base class altogether (BaseDCCIntegrator) and replace it with an interface, then take any functionality that the base class has and share it via a helper class.  I have documented this possible improvement here: https://trello.com/c/qRDdO1sC